vocabulary-draft/continuous-integration-pipeline-events.md

```diff
@@ -0,0 +1,28 @@
+# Continuous Integration Events
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+These events are related to **Continuous Integration(CI)** activities. CI usually include activities such as building, testing, packaging and releasing software artifacts. 
```

Comment:
Personally I don't see that "releasing" has anything to do with CI, rather with Continuous Delivery. Maybe what you mean by releasing is actually "publishing", i.e. pushing an artifact to some artifacts repository? To me "releasing" is a more business oriented activity that often happens very late in a CI/CD chain.